### PR TITLE
Fix/faq text update

### PR DESCRIPTION
**Pull request by kfilip94**
This PR adds some text revisions and increase space in the first faq header (between `is` and `Climate`)
[PIVOTAL](https://www.pivotaltracker.com/story/show/164532347)

**BEFORE:**
Screenshot from 2019-03-15 12 19 44
**AFTER:**
Screenshot from 2019-03-15 12 19 30

Text revisions file:
[FAQ_kelly comments ce.docx](https://github.com/Vizzuality/climate-watch/files/2970974/FAQ_kelly.comments.ce.docx)

TODO: set table dynamic height and capitalization of column headers

### CODE REVIEW

**Comment by weberjavi** [comment]
☝️ on it

File: app/javascript/app/components/about/about-faq/about-faq-data.js [hunk 1]
```diff
@@ -192,20 +192,20 @@ export const sectionsData = [
...
+          'According to the Working Group III Contribution to the IPCC Fifth Assessment Report, global CO<sub>2</sub> emissions from fossil fuel combustion are known within 8% uncertainty (90% confidence interval). CO<sub>2</sub> emissions from FOLU (forestry and other land use) have very large uncertainties associated with them in the order of ± 50%. Uncertainty for global emissions of CH<sub>4</sub>, N<sub>2</sub>O and the F-gases has been estimated as 20%, 60% and 20% respectively. For the PIK dataset, regional deforestation emissions are downscaled to the country level using estimates of the deforested area obtained from potential vegetation and calculations for the needed agricultural land. Accordingly, levels of uncertainty are quite high for earlier years of data (closer to 1850).  '
```
Comment (weberjavi) [inline app/javascript/app/components/about/about-faq/about-faq-data.js]: On point three of `Historical emissions module` FAQ, in the last line, there's a coma after `Accordingly, levels...` that is not present on the corrections document.
image

File: app/javascript/app/components/about/about-faq/about-faq-data.js [hunk 2]
```diff
@@ -284,9 +284,9 @@ export const sectionsData = [
       },
       {
         type: 'text',
-        title: 'Why is data missing for some countries? ',
+        title: 'Why are data missing for some countries? ',
```
Comment (weberjavi) [inline app/javascript/app/components/about/about-faq/about-faq-data.js]: This change (`is` for `are`) is not reflected in the document

Comment (kfilip94) [reply app/javascript/app/components/about/about-faq/about-faq-data.js]: @weberjavi Hmm, I think it is:
Screenshot from 2019-03-15 14 11 11

File: app/javascript/app/components/about/about-faq/about-faq-data.js (hunk 1 above)
Comment (kfilip94) [reply app/javascript/app/components/about/about-faq/about-faq-data.js]: Thanks, fixed!

**Pull request merged**